### DISCUSSION TITLE
Error while deploying new project

### DISCUSSION THREAD

**Discussion by ThewBear** [vercel/vercel#5110] [General] [answered]
When I run `vc` to link and deploy new project, I get the following error.

```bash
Vercel CLI 20.0.0
? Set up and deploy “~\myproject”? [Y/n] y
? Which scope do you want to deploy to? myname
? Link to existing project? [y/N] n
? What’s your project’s name? myproject
? In which directory is your code located? ./
No framework detected. Default Project Settings:
- Build Command: `npm run vercel-build` or `npm run build`
- Output Directory: `public` if it exists, or `.`
- Development Command: None
? Want to override the settings? [y/N] n
Error! Invalid vercel.json - `projectSettings` should NOT have additional property `sourceFilesOutsideRootDirectory`. Please remove it.
View Documentation: https://vercel.com/docs/configuration#project/projectsettings
```

However, if I link the project first with `vc link` then deploy with `vc`, it deployed successfully.

**Comment by ThewBear** [comment]
Related to #5109

**Comment by ThewBear** [comment, marked as answer]
Might be related to the new feature https://vercel.com/blog/monorepos. Closing this for now.

**Comment by Afoma** [comment]
I encounter this error after entering my project's name
" What’s the name of your existing project? 
Error! Project not found "

**Comment by Afoma** [comment]
I encounter this error after entering my project's name
" What’s the name of your existing project?
Error! Project not found "
How do I fix it?

**Comment by abdurrauf2022** [comment]
Error! Project not found
? What’s the name of your existing project?

**Reply by amyegan** [reply]
Hi @abdurrauf2022! Please [start a new discussion](https://github.com/vercel/vercel/discussions/new) with details about your project and the steps taken that lead to the error. I would be happy to help you find a solution.

**Reply by abdurrauf2022** [reply]
ok

**Comment by abdurrauf2022** [comment]
how to fix it pls.

**Comment by abdurrauf2022** [comment]
how to fix... on visual studio code
Error! Project not found
? What’s the name of your existing project?

**Comment by dimad3** [comment]
J have the same issue:
```
Error! Project not found
? What’s the name of your existing project?
```
How to fix it, please!

**Reply by amyegan** [reply]
Hi @dimad3. Can you [start a new discussion](https://github.com/vercel/community/discussions/new) with more information about your project, the steps leading to this error, and any additional error details you have available? I think your issue is slightly different from what is happening in the original post here.

**Comment by priyankasaini69** [comment]
I encounter this error after entering my project's name
" What’s the name of your existing project?
Error! Project not found "
How do I fix it?

**Reply by amyegan** [reply]
Hi @priyankasaini69! If you're getting this error after following the [Creating & Deploying Your Project](https://vercel.com/docs/concepts/get-started/deploy#) instructions, please [start a new discussion](https://github.com/vercel/community/discussions/new) with more information about your project 🙏
